### PR TITLE
Realm CLI tool

### PR DESCRIPTION
**Pull request by clementetb**
It is a convenient single entry point to the realm helper tools.

It allows to control the sync server:
- Start
- Stop
- Restart
- Status
- Bind emulator ports

And the realm-java library:
- install locally
- build
- test (run full test suite as in Jenkins)
- check
- clean

### CODE REVIEW

**Comment by edualonso** [comment]
This is pure gold, thanks!

**Review by rorbech — COMMENTED** [review]
Should probably be renamed to avoid confusion with https://docs.mongodb.com/realm/deploy/realm-cli-reference/

**Review by edualonso — APPROVED** [review]

File: tools/realm-cli.sh [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+case $command in
+    server)
+    server $*
+    ;;
+    java)
+    java $*
+    ;;
+    *)
+    show_help
+esac
```
Comment (cmelchior) [inline tools/realm-cli.sh:185]:
```suggestion
esac

```

**Review by cmelchior — APPROVED** [review]
Pretty cool 👏 . Like Claus mentioned, `realm-cli` is already used by the MongoDB Realm CLI tool, but I guess that is solved by just giving it another alias in your bash config, so 👍

**Pull request merged**